tox.ini

```diff
@@ -15,7 +17,6 @@ commands =
     {envbindir}/pytest
 
 [testenv:lint]
-basepython = python3
```

Comment:
Please keep this line.

Reply:
This really is a no-op nowadays, since the only supported Python is v3.